docs-source/content/userguide/managing-domains/domain-lifecycle/restarting.md

```diff
@@ -3,116 +3,123 @@ title: "Restarting"
-      * If you change other fields that typically cause a restart, such as `volumes`, `env`, and such, then the introspector job will not rerun and a rolling restart will proceed without loading the configuration changes.
+    * If you are supplying updated models or Secrets for a running domain, and you want the configuration updates to take effect using a rolling restart, then do one of the following:
+      * Supply a new value for the `image` field in the Domain or any of the other [fields affecting WebLogic Server instance Pod generation]({{< relref "/userguide/managing-domains/domain-lifecycle/startup/_index.md#fields-that-cause-servers-to-be-restarted" >}}).
+      * Change the Domain `restartVersion` field. This will cause the operator to restart all running servers and, prior to the restarts, the operator will introspect any new configuration.
+      * Change the Domain `introspectVersion` field. This will cause the operator to introspect any new configuration and, if needed, restart servers to use that new configuration.
 
 * **Domain in PV:**
-For a domain home on PV, the type of restart needed to apply the changes depends on the nature of the WebLogic configuration change:
-    * Changes to parts of the WebLogic configuration that the operator introspects, require a full restart, even if the changes are dynamic.
-      The following are the types of changes to the WebLogic Server configuration that the operator introspects:
-        * Adding or removing a cluster, server, dynamic server, or network access point
+
+For Domain in PV, the type of restart needed depends on the nature of the WebLogic domain configuration change:
+    * With operator version 3.0.0, domain configuration changes that add clusters, either configured or dynamic, cluster member servers, or standalone servers can now be performed dynamically. This support requires that the new clusters or servers are added to the domain configuration and then [triggering the operator's introspection]({{< relref "/userguide/managing-domains/domain-lifecycle/introspection.md" >}}) of that new configuration.
+    * Other changes to parts of the domain configuration that the operator introspects, require a full restart, even if the changes are dynamic.
+      The following are the other types of changes to the domain configuration that the operator introspects:
+        * Adding or removing a network access point
         * Changing a cluster, server, dynamic server, or network access point name
         * Enabling or disabling the listen port, SSL port, or admin port
         * Changing any port numbers
         * Changing a network access point's public address
-    * Other dynamic WebLogic configuration changes do not require a restart.  For example, a change to a server's connection timeout property
+    * Other dynamic WebLogic configuration changes do not require a restart. For example, a change to a server's connection timeout property
```

Comment:
Prepend "With operator version 3.0.0, ..." just like you did for the first bullet.

Reply:
Also, perhaps this bullet, and the next, can be grouped with the first bullet.

Reply:
It looks like this comment has not been addressed yet.

Reply:
I went the other way and made the "it didn't always work like this" a note after.